<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>681</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>100</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Seleccionar Algoritmo:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>210</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ejemplares cereales y derivados:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>280</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>190</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>310</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ejecutar</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>320</y>
        <width>371</width>
        <height>241</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>230</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ejemplares leche, huevo y derivados:</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>471</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>944</x>
        <y>600</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>Fuente de textos (organizados por subcarpetas)</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>230</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>360</y>
        <width>511</width>
        <height>201</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <row>
       <property name="text">
        <string>Carne</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Cereales</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Leche</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Legumbres</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Carne</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cereales</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Leche</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Legumbres</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="0" column="3">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="1" column="3">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="2" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="2" column="3">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="3" column="2">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="3" column="3">
       <property name="text">
        <string>0</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>280</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Total:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>30</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Abrir</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>250</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>100</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>190</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ejemplares carne, pescado, frutas y verduras:</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>100</y>
        <width>401</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit_5">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>600</y>
        <width>511</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>210</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>250</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ejemplares legumbres:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>600</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Guardar modelo:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>370</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Resultado:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>160</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Vista Previa:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>150</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Modelo clasificador:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cpushButton_3">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>220</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ejecutar</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="ctableWidget">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>290</y>
        <width>481</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <column>
       <property name="text">
        <string>Texto</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Categoria</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ver texto</string>
       </property>
      </column>
     </widget>
     <widget class="QPlainTextEdit" name="cplainTextEdit">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>90</y>
        <width>461</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="cpushButton_2">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>150</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Abrir</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>250</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Resultado:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cpushButton">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>90</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Abrir</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="cplainTextEdit_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>150</y>
        <width>461</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>90</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Textos a clasificar:</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="cplainTextEdit_3">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>580</y>
        <width>471</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="cpushButton_4">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>580</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>580</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Guardar resultados:</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="cplainTextEdit_4">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>290</y>
        <width>481</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>250</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Texto seleccionado:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QTableWidget" name="stableWidget">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>150</y>
        <width>741</width>
        <height>401</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <column>
       <property name="text">
        <string>Carrefour</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Dia</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="spushButton_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>90</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Comparar precios</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="splainTextEdit">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>30</y>
        <width>471</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="spushButton">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>30</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Abrir</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>570</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Precio minimo posible:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>570</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>?</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>600</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Supermercado mas barato:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>600</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>?</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Texto a comparar:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
